docs/navigation-parameters-reference.md

```diff
@@ -151,8 +151,8 @@ You can configure the way Luigi tackles routing in your application in the **Rou
 
 The node navigation parameters are as follows:
 
-- **nodeAccessibilityResolver** allows you to define a permission checker function that gets executed on every node. If it returns `false`, Luigi removes the node and its children from the navigation structure.
-- **nodeAccessibilityResolver** receives all values defined in the node configuration. See [angular basicConfiguration.js](../core/examples/luigi-sample-angular/src/assets/basicConfiguration.js) for the **constraints** example.
+- **nodeAccessibilityResolver** receives all values defined in the node configuration. It allows you to define a permission checker function that gets executed on every node. If it returns `false`, Luigi removes the node and its children from the navigation structure.
```

Comment:
I combined these two together. I also changed the link to extended configuration because actually there wasn't much about it in the basic example  🤔  Is it correct? Or should it be the basic one?

Reply:
What you did is correct :)